src/main.ts

```diff
@@ -63,8 +64,11 @@ async function startVue () {
         'ResizeObserver loop limit exceeded',
         'ResizeObserver loop completed with undelivered notifications',
         // ignore Launch Darkly errors (partial match)
-        'LD: [error]'
-      ]
+        'LD: [error]',
+        // ignore Vuex errors (partial match)
+        '[vuex]'
```

Comment:
I removed this vuex ignore line in a future commit, as I think we should report this in Prod. But I'm adding a comment to this commit because I found something interesting...

While testing the Release tag in Sentry, VS Code sent a patch to my browser, which output a ton of these "[vuex] duplicate getter" errors, followed by a bunch of "undefined this" and "undefined that" errors -- exactly what we see it Prod. So I have high hopes that the vuex module refactor will fix a ton of these errors!!!

(I don't know how users are getting these errors, since they're not using a debugger/patches in Prod, but the symptoms are the same so I expect the root cause is the same, too.)